member-load-test.js

```
@@ -0,0 +1,630 @@
+
+export function teardown() {
+    console.log('='.repeat(70));
+    console.log('Member API 종합 기능 테스트 완료');
+    console.log(`총 Member API 에러: ${memberApiErrors ? memberApiErrors.count || 0 : 0}`);
+
+    // 테스트 계정 일괄 삭제
+    try {
+        let cleanupResponse = http.del(`${BASE_URL}/api/test/auth/cleanup`);
+        console.log(`정리 작업 완료: ${cleanupResponse.status}`);
+    } catch (e) {
+        console.log('정리 작업 중 오류:', e.message);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)